forem/forem
# Fix: Billboard dropdown scrolling within the billboard

## What type of PR is this? (check all applicable)


## Description
In some cases (where the height is shorter than the billboard dropdown menu) like on the sidebars of the home page and article pages, when we click on a billboard three dot menu, it starts scrolling within the billboard.

We also see the billboard dropdown contained within the billboard when the width of the dropdown is bigger then the billboard.

<img width="384" alt="Screenshot 2023-07-21 at 14 25 34" src="https://github.com/forem/forem/assets/2786819/512392eb-f562-47e4-979d-67e077b9e812">

<img width="535" alt="Screenshot 2023-07-21 at 14 26 11" src="https://github.com/forem/forem/assets/2786819/97ea3399-f5ae-44a5-ba2e-4192750599ae">

## Related Tickets & Documents

- Related Issue #19804 
- Closes #19804

## QA Instructions, Screenshots, Recordings

Go to any sidebar billboard and open the dropdown. It should show the full menu and not scroll within the billboard. It should also not cut off. Test other billboards as well. 

https://www.loom.com/share/56d00e77330e40c19d8f86aa571e259c?sid=57c02c7f-049c-44ac-a9f9-3934b44737ea

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_

_For more info, check out the
[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._

## Added/updated tests?

- [x] No, I didn't write a test for the JS to implement the style because its amending the DOM styles rather than logic, but I can try to if folks feel differently

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](https://media.giphy.com/media/2x0VePimPaFJDpGZ7H/giphy-downsized-large.gif)
